1_Diversity-Participation.md

```diff
@@ -0,0 +1,28 @@
+# Metric: Diversity and Participation
+
+Activity Metric | Description
+--- | ---
+New Contributors vs Maintainers | Ratio of new contributors to maintainers over time

```

Comment:
Is there any difference between "contributors" and "maintainers"? If so, I think we would need a definition. If not, I suggest to use the same word everywhere (likely, "contributors"), to avoid confusion.

Reply:
How about:
- Contributor = Anyone who provides a patch, pull request, wiki change, or other contribution.
- Maintainer = Contributor who has commit rights to the main branch and can merge contributions from others.

Reply:
It is not always easy to know who has those rights... And  much less to know that over time (for comparing with the past, for example).

Reply:
I agree that this information may not be easily known for third-party projects. It may be more easily known if you are involved in the project. Thus, the metric might have value for assessing internal projects. We also wanted to capture metrics that are meaningful and then hear from the software committee if can be implemented or needs unavailable data sources.